tests/test_sdss.py

```diff
@@ -22,6 +30,7 @@ def test_sdss(self, cfg):
         assert an_obj["gain"][3] == pytest.approx(4.76)
         assert isinstance(an_obj["wcs"], list)
 
+        # fake download 745 and just check that the size is right
```

Comment:
as per discussion, should we just make 745 available in repo / Git LFS? And remove the monkeypatch overriding code above? I think it'll just be cleaner, and we can use the 2-field dataset in future.

Reply:
I'm just going to remove that second test for multiple fields for now. Since the downloader is already kind of sketchy, we should update this to have more in-depth tests that also test downloader functionality when we fix that.